File: src/i18n/en_nz.go
package i18n

// enNZOverrides is empty for the same reason as enAUOverrides: New Zealand
// English follows British spelling, which inheriting en-GB already supplies,
// and its distinctive vocabulary lies outside anything a storage and service
// control panel says.
var enNZOverrides = map[string]string{}

// enNZMessages is en-GB unchanged, reviewed for New Zealand usage.
var enNZMessages = derive(enGBMessages, enNZOverrides)

File: ui/src/i18n/en-NZ.js
import derive from './derive.js'
import enGB from './en-GB.js'

/**
 * New Zealand English follows British spelling, supplied by inheriting en-GB.
 * Its distinctive vocabulary lies outside anything a storage and service
 * console says.
 *
 * Empty is the finding, not a gap.
 */
export const enNZOverrides = {}

/** English (New Zealand) — en-GB unchanged, reviewed for New Zealand usage. */
const enNZ = derive(enGB, enNZOverrides)

export default enNZ
